themes/classic/modules/blockcart/modal.tpl

```diff
@@ -34,7 +34,10 @@
               {/if}
               <p><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.products.value}</p>
               <p><strong>{l s='Total shipping:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.shipping.value}</p>
-              <p><strong>{l s='Total:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.totals.total.value}</p>
+              {if isset($cart.subtotals.tax.label)}
```

Comment:
Can you replace by `{if $cart.subtotals.tax}`?
We want to avoid the use of `isset` as soon as it's possible.